low-level/sspl-ll.spec

```diff
@@ -140,6 +142,13 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 [ -f $SSPL_DIR/sspl_init ] ||
     ln -s $SSPL_DIR/bin/sspl_provisioner_init $SSPL_DIR/sspl_init
 
+# Creating softlink under site-packages to use sspl module easier way.
+[ -d "${SSPL_DIR}" ] && {
+    mkdir -p /usr/lib/python3.6/site-packages/cortx/sspl
```

Comment:
define path in a var & use it. Also what if /usr/lib/python3.6/site-packages/cortx/sspl dir exists, should just clean it up & proceed then.